frontend/appflowy_flutter/lib/mobile/application/mobile_theme_data.dart

```diff
@@ -1,45 +1,82 @@
 // ThemeData in mobile
 import 'package:flutter/material.dart';
 
-ThemeData getMobileThemeData() {
-  const mobileColorTheme = ColorScheme(
-    brightness: Brightness.light,
-    primary: Color(0xFF2DA2F6), //primary 100
-    onPrimary: Colors.white,
-    // TODO(yijing): add color later
-    secondary: Colors.white,
-    onSecondary: Colors.white,
-    error: Color(0xffFB006D),
-    onError: Color(0xffFB006D),
-    background: Colors.white,
-    onBackground: Color(0xff2F3030), // title text
-    outline: Color(0xffBDC0C5), //caption
-    //Snack bar
-    surface: Colors.white,
-    onSurface: Color(0xff2F3030), // title text
-  );
+const _primaryColor = Color(0xFF2DA2F6); //primary 100
+const _onBackgroundColor = Color(0xff2F3030); // text/title color
+const _onSurfaceColor = Color(0xff676666); // text/body color
+const _onSecondaryColor = Color(0xFFC5C7CB); // text/body2 color
+
+// TODO(yijing): improve theme color before release
+ThemeData getMobileThemeData(Brightness brightness) {
+  final mobileColorTheme = (brightness == Brightness.light)
+      ? ColorScheme(
+          brightness: brightness,
+          primary: _primaryColor,
+          onPrimary: Colors.white,
+          // TODO(yijing): add color later
+          secondary: Colors.white,
+          onSecondary: _onSecondaryColor,
+          error: const Color(0xffFB006D),
+          onError: const Color(0xffFB006D),
+          background: Colors.white,
+          onBackground: _onBackgroundColor,
+          outline: const Color(0xffBDC0C5), //caption
+          outlineVariant: const Color(0xffCBD5E0).withOpacity(0.24),
+          //Snack bar
+          surface: Colors.white,
+          onSurface: _onSurfaceColor, // text/body color
+        )
+      : ColorScheme(
+          brightness: brightness,
+          primary: _primaryColor,
+          onPrimary: Colors.white,
+          // TODO(yijing): add color later
+          secondary: Colors.black,
+          onSecondary: Colors.white,
+          error: const Color(0xffFB006D),
+          onError: const Color(0xffFB006D),
+          background: const Color(0xff1C1C1E), // BG/Secondary color
+          onBackground: Colors.white,
+          outline: const Color(0xff96989C), //caption
+          outlineVariant: Colors.black,
+          //Snack bar
+          surface: const Color(0xff2F3030),
+          onSurface: const Color(0xffC5C6C7), // text/body color
+        );
+
   return ThemeData(
     // color
     primaryColor: mobileColorTheme.primary, //primary 100
     primaryColorLight: const Color(0xFF57B5F8), //primary 80
     dividerColor: mobileColorTheme.outline, //caption
+    hintColor: mobileColorTheme.outline,
+    disabledColor: mobileColorTheme.outline,
     scaffoldBackgroundColor: mobileColorTheme.background,
     appBarTheme: AppBarTheme(
       foregroundColor: mobileColorTheme.onBackground,
       backgroundColor: mobileColorTheme.background,
-      elevation: 0,
+      elevation: 80,
```

Comment:
This is a huge number for `elevation` in Flutter's Material Widgets, the default value in an AppBar is `4.0`, so `80` seems quite exaggerated... :eyes:

Reply:
I prefer to use a divider to separate the appbar and body.